### ISSUE TITLE
Can't restart broadcasting or apply changes

### ISSUE THREAD

**Issue by jolo22** [AzuraCast/AzuraCast#1888]
When I click "Restart Broadcasting" inside utilities or "Restart to Apply Changes". It will just stuck on "please wait..." and nothing is happening.

When I copy and paste the link on my browser: 
http://983radio.ddns.net/api/station/1/restart

The message stated:
500
Method not allowed.
RoutingMiddleware.php : 94

I don't know what happened, I just freshly installed this one and added music. It doesn't happen to me in the past.

**Comment by jolo22** [comment]
![azuraprobs](https://user-images.githubusercontent.com/15798127/63346633-ff231000-c387-11e9-835b-b7ffc1c01ff2.PNG)
More evidence of the problem.

**Comment by SlvrEagle23** [comment]
@jolo22 The part about not being able to call the `restart` API command from your browser makes sense; it only accepts the POST method, not the GET method which your browser would normally be sending.

On that web inspector panel, switch over to "Network", try the restart again, and then click on the red request line that appears, then click "Response". Copy that response body and paste it here so we can diagnose the issue further.

**Comment by jolo22** [comment]
![resultazura](https://user-images.githubusercontent.com/15798127/63377562-f6036480-c3c2-11e9-84b1-6ee82d8e5b57.PNG)
Here is the result

**Comment by SlvrEagle23** [comment]
@jolo22 Oh. Hmm.

Let's try the other way around: try restarting the station via the sidebar, and then go visit the system log viewer (in System Administration) and click the "AzuraCast Log", and see if it's producing any error messages that would be relevant.

**Comment by jolo22** [comment]
Is it the "Azuracast Application Log"? I clicked on to that and when I click "Restart Broadcasting", It doesn't show the error in the logs. It's like nothing is happening when clicked. Instead I just got this error:

[2019-08-20 20:07:24] AzuraCast.WARNING: Method not allowed. {"file":"/var/azuracast/www/vendor/slim/slim/Slim/Middleware/RoutingMiddleware.php","line":94,"code":405} []

In which I got by accessing the previously stated link through my browser. But I think this error is not relevant since like what you said, it only accepts the POST method.
